ruby/lib/minitest/queue.rb

```diff
@@ -83,6 +94,11 @@ def queue_reporters=(reporters)
       @queue_reporters = reporters
     end
 
+    attr_reader :flaky_test_supplier
+    def flaky_test_supplier=(flaky_test_supplier)
```

Comment:
Why not a simple `attr_accessor` ?

Reply:
Also I'd rather access `queue.flaky_test...` than increase the minitest patch surface.

We can even facade this with `queue.flaky?(test)`.